process/block/preprocess/interfaces.go

```diff
@@ -15,7 +15,7 @@ type SortedTransactionsProvider interface {
 
 // TxCache defines the functionality for the transactions cache
 type TxCache interface {
-	SelectTransactionsWithBandwidth(numRequested int, batchSizePerSender int, bandwidthPerSender uint64) []*txcache.WrappedTransaction
+	SelectTransactions(gasRequested uint64, maxNum int) ([]*txcache.WrappedTransaction, uint64)
```

Comment:
See `TxCacheSelectionGasRequested` and `TxCacheSelectionMaxNumTxs` in `process/constants.go`.